src/bp/ui-shared/src/typings.d.ts

```diff
@@ -25,7 +25,7 @@ import { FormField, MultiLangText } from 'botpress/sdk'
 import { MoreOptionsProps } from '../../ui-shared-lite/MoreOptions/typings'
 import { OverlayProps } from '../../ui-shared-lite/Overlay/typings'
 import { ToolTipProps } from '../../ui-shared-lite/ToolTip/typings'
-
+import { Item } from './FormFields/TagInputList/index'
```

Comment:
Item is too generic and it doesn't fit well with other typings. What is it exactly? Also, don't add "index"

Reply:
Item is the type of one item in the enumeration. I don't mind redefining it in the EnumForm because it's really a small object